server/src/controllers/Events/resolver.ts

```diff
@@ -657,7 +658,12 @@ ${unsubscribe}`;
 
     const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter?.id}`;
     const eventURL = `${process.env.CLIENT_LOCATION}/events/${event.id}?emaillink=true`;
-    // TODO: this needs to include an ical file
+    const calendar = ical().createEvent({
+      start: event.start_at,
```

Comment:
```suggestion
    const calendar = ical();
    calendar.createEvent({
      start: event.start_at,
```
Turns out `createEvent` returns *just* that event, rather than the original calendar.  Unfortunately Google calendar won't import single events, only entire calendars :man_shrugging:

Reply:
Sorry about that. I had it on separate lines initially, and when I tested it :man_facepalming:.

Reply:
No worries, it *looks* like it should be a fluent interface, but apparently not :(